src/renderer/components/AccountView.tsx

```diff
@@ -37,28 +38,64 @@ const explorerURL = (net: Net, address: string) => {
   }
 };
 
-function AccountView(props: { pubKey: string | undefined }) {
+function AccountView(props: { pubKey: string }) {
   const { pubKey } = props;
   const { net } = useAppSelector(selectValidatorNetworkState);
 
-  const [account, setSelectedAccountInfo] = useState<AccountInfo | undefined>(
-    undefined
-  );
+  if (!pubKey) {
```

Comment:
You can't reassign this cause it was initialized as `const`, not `let`. My VSCode stack warns me about this, did yours not?

Reply:
yeah, i agree, v odd.